### ISSUE TITLE
Failed to download assembly_summary.txt

### ISSUE THREAD

**Issue by schmittel** [pirovc/genome_updater#84]
Hi, thanks for the update.

I'm getting a strange error when downloading the latest GTDb:

```
-------------------------------------------
┌─┐┌─┐┌┐┌┌─┐┌┬┐┌─┐    ┬ ┬┌─┐┌┬┐┌─┐┌┬┐┌─┐┬─┐
│ ┬├┤ ││││ ││││├┤     │ │├─┘ ││├─┤ │ ├┤ ├┬┘
└─┘└─┘┘└┘└─┘┴ ┴└─┘────└─┘┴  ─┴┘┴ ┴ ┴ └─┘┴└─
                                     v0.6.1 
-------------------------------------------
Mode: NEW 
Args: -M 'gtdb' -t '10' -o '/genomes/gtdb_r214' -g 'archaea,bacteria' -f 'genomic.fna.gz' -d 'refseq,genbank' -m
Outp: /genomes/gtdb_r214/
-------------------------------------
Downloading assembly summary [2023-04-29_13-08-18]
 - Database [refseq,genbank]
 - Organism group [archaea,bacteria]
 - Failed to download ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/archaea/assembly_summary.txt. Trying again #2
 - Failed to download ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/archaea/assembly_summary.txt. Trying again #3
 - Failed to download one or more assembly_summary files
 ```

I'm able to directly access `ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/archaea/assembly_summary.txt` so I'm not sure what the issue is here. 

Many thanks for your help

**Comment by aziele** [comment]
Hi,

I'm getting the exact same error message returned by the genome_updater tool (`Failed to download ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/archaea/assembly_summary.txt`). Interestingly, I can successfully download the file using either wget or curl from the command line.

Unfortunately, the suggestions provided by @pirovc to resolve the issue did not work in my particular case. I noticed that this problem started in genome_updater v.0.5.0 and still persists, whereas everything works find in version 0.4.1.

Thanks for your help!

**Comment by SebaSilvaS** [comment]
Hello! I am using the v0.6.2 on a linux machine and I am running into the same error. 

I tried `retries=10 timeout=600 ./genome_updater.sh -d "refseq,genbank" -g "bacteria" -f "genomic.fna.gz" -o "MAGs/Acidimicrobiia_MAGs" -M "gtdb" -T "c__Acidimicrobiia" -t 8 -R 5 -L curl`

Any ideas?

**Comment by pirovc** [comment]
This is a bug reported in #86, a fix will be provided today.

**Comment by pirovc** [comment]
Fixed with v0.6.3 #87
